resubmit.py

```diff
@@ -49,33 +50,30 @@ def modifySchema(helper, workflow, user, group, cache, events, firstLumi, backfi
     paramBlacklist = ['BlockCloseMaxEvents', 'BlockCloseMaxFiles', 'BlockCloseMaxSize', 'BlockCloseMaxWaitTime',
```

Comment:
Yep, InputDatasetS was removed. RequestNumEvents sort of depends on the request type. But in the end it relies on whether there is InputDataset or not.